src/camera.rs

```diff
@@ -71,33 +82,38 @@ impl CamBuilder {
 
     pub fn build(&self) -> Camera {
         let dir = self.center - self.eye;
+        let y = Vec3::new(0.0, 0.0, 1.0).dot(&dir).acos().to_degrees();
         Camera {
             eye: self.eye,
             dir: dir,
             up: self.up,
             fov: self.fov,
             width: self.width,
-            height: self.height
+            height: self.height,
+            angles: Rotate { x: 0.0, y: y },
```

Comment:
afaik if the variable name is the same as the field name, you can omit the field name
```suggestion
            angles: Rotate { x: 0.0, y },
```